### PR TITLE
400 don t provide content length

### PR DESCRIPTION
**Pull request by Natim**
r? @n1k0

### CODE REVIEW

File: kinto/tests/test_default_bucket.py [hunk 1]
```diff
@@ -56,9 +56,10 @@ def test_second_call_on_default_bucket_collection_doesnt_raise_a_412(self):
         self.app.get(self.collection_url, headers=self.headers)
 
     def test_querystring_parameters_are_taken_into_account(self):
-        self.app.get(self.collection_url + '/records?_since=invalid',
-                     headers=self.headers,
-                     status=400)
+        resp = self.app.get(self.collection_url + '/records?_since=invalid',
+                            headers=self.headers,
+                            status=400)
+        self.assertIn('Content-Length', resp.headers)
```
Comment (n1k0) [inline kinto/tests/test_default_bucket.py]: I wonder if we should create new tests for header checks; here for instance we're mixing querystring and headers. Thoughts?

Comment (Natim) [inline kinto/tests/test_default_bucket.py]: I was just trying to debug, but I suppose you are right. I will had a test that does the right thing (checking for Allow-Control-Expose-Headers)

**Comment by Natim** [comment]
I have created https://github.com/mozilla-services/cliquet/pull/436 but I cannot reproduce. Have we got this fixed in last Cliquet version?

**Comment by leplatrem** [comment]
r- : I would not spread the same assertions like this everywhere !

An assertion in a test should reflect its title.

**Comment by Natim** [comment]
Actually we do not plan to land this

**Pull request closed without merging**